### ISSUE TITLE
Fix Student Assessment results that appear off by 1 day in the UI

### ISSUE THREAD

**Issue by alexsoble** [studentinsights/studentinsights#622]
```
# Find most recently taken DIBELS student assessment:
dibels = Assessment.find_by_family('DIBELS').student_assessments.order(:date_taken).last

# Check date taken
dibels.date_taken
=> Fri, 01 Jan 2016

# Look up student
dibels.student

# 2016 DIBELS result not appearing on student profile! # 
```

**Comment by alexsoble** [comment]
The record is in the database, just not making it to the UI. Going to throw it in a bunch of debugging statements and see why this isn't showing up.

**Comment by alexsoble** [comment]
```
puts student.dibels.order(:date_taken).pluck(:date_taken).reverse

=>
2016-01-01 00:00:00 UTC
2015-10-01 00:00:00 UTC
2015-05-01 00:00:00 UTC
2015-01-01 00:00:00 UTC
2014-10-01 00:00:00 UTC
```

**Comment by alexsoble** [comment]
Dates are showing up in the UI off by 1 day. 

For example:

```
# db
2016-01-01 00:00:00 UTC

# UI
December 31st, 2015
```

**Comment by alexsoble** [comment]
# The Problem

```
# js

moment("2016-01-01T00:00:00.000Z").toDate()
=> Thu Dec 31 2015 18:00:00 GMT-0600 (CST)

moment("2016-01-01").toDate()
=> Fri Jan 01 2016 00:00:00 GMT-0600 (CST)
```

**Comment by alexsoble** [comment]
This commit fixes the issue for DIBELS: 3e9442b88f25af2115158558a0fd782730fcdff4

Next steps are:

(A) See if this issue is affecting other types of assessments 

and

(B) Plan out a more generalized solution if needed. 

Not sure right now if it's best to handle this client-side or server-side.

**Comment by alexsoble** [comment]
This issue exists for STAR assessments in the student Full Case History (but interestingly not the line graphs).

**Comment by alexsoble** [comment]
This issue exists for MCAS assessments in the student Full Case History (but also not in the line graphs.)

**Comment by alexsoble** [comment]
Checking to see how JS date is computed for the line graphs as opposed to the Full Case History on the Student Profile . . .

**Comment by alexsoble** [comment]
The dates that look right are coming from serialized_data/chart_data.

**Comment by alexsoble** [comment]
Profile dates STAR, MCAS, etc are being computed using the `QuadConverter`; this is improperly returning dates that are one day early. 

```
// js 

QuadConverter.toMoment([2015, 2, 5, 0]);
> Moment {_isAMomentObject: true, _i: "2015-2-5", _f: "YYYY-M-D", _isUTC: true, _pf: Object…}
```

```
// js

QuadConverter.toMoment([2015, 2, 5, 0]).toDate();
> Wed Feb 04 2015 18:00:00 GMT-0600 (CST)
```

**Comment by alexsoble** [comment]
Looks like a timezone issue. Moment.js is subtracting 6 hours from the date in order to return a time in CST. We need to stop it from subtracting those six hours.

**Comment by alexsoble** [comment]
Fixed by #635.
